apps/web/src/pages/details/[applicantId].tsx

```diff
@@ -123,8 +118,8 @@ const Details = () => {
           <FontAwesomeIcon className='h-5 mr-2 text-blue-900 ' icon={faClock}></FontAwesomeIcon>
           <h1 className='text-xl text-blue-900 py-4 font-bold'>Milestones Logs</h1>
         </div>
-        <HeaderTab tabs={milestoneTabs} onTabClick={onTabClick} />
-        {currentTab === '' ? <h1>Intake</h1> : currentTab}
+        <HeaderTab tabs={milestoneTabs} tabIndex={currentTab} onTabClick={setCurrentTab} />
+        {logType[currentTab]?.component}
```

Comment:
I think this will cause the tab being shown to be 1 more than the correct one.  ex. clicking on Intake will show Licensing/Registration

Reply:
because the header values are from 1-5, but the logType array is from 0-4